UserInfo userInfo
geometry_msgs/Point32 centerOfMassImage
geometry_msgs/Point32 centerOfMassWorld
RectWithConfidence userRect
RectWithConfidence headBoundingBox
sensor_msgs/Image geodesicMatrix
LandmarksInfo landmarksInfo
SkeletonJoint[] skeletonJoints
Gestures gestures

